packages/core/test/guards.test.ts

```diff
@@ -178,19 +178,22 @@ describe('guard conditions', () => {
               T1: [
                 {
                   target: 'B1',
-                  cond: (_state, _event, { state: s }) => s.matches('A.A1')
+                  cond: (_state: any, _event: any, { state: s }: any) =>
```

Comment:
Conditional transitions are acting weird when no generics are provided. Even adding something like this fixes the problem:
```js
schema: {
  context: {} as unknown
}
```

This could be investigated in the future.